tests/encodehelp.h

```diff
@@ -152,13 +155,16 @@ static bool process_cmdline(int argc, char *argv[])
                     rcMode = string_to_rc_mode(optarg);
                     break;
                 case 3:
-                    ipbMode= atoi(optarg);
+                    ipbMode = atoi(optarg);
```

Comment:
This changed by git-format-clang,right? Instead of format your file you can format your patch only. git-format-clang --sytle=webrtc HEAD^. Use this approach we can find who is really change the code by "git blame"